pkg/controller/container-runtime-config/helpers.go

```diff
@@ -28,10 +27,12 @@ import (
 const (
 	minLogSize           = 8192
 	minPidsLimit         = 20
-	crioConfigPath       = "/etc/crio/crio.conf"
 	storageConfigPath    = "/etc/containers/storage.conf"
 	registriesConfigPath = "/etc/containers/registries.conf"
 	policyConfigPath     = "/etc/containers/policy.json"
+	// CRIODropInFilePath is the path at which changes to the crio config will be dropped in
+	// this is exported so that we can use it in the e2e-tests
+	CRIODropInFilePath = "/etc/crio/crio.conf.d/01-newconfig"
```

Comment:
can we name this something like `01-ctrcfg` or something more informative of its source?

Reply:
fixed